wallet/src/main/AndroidManifest.xml

```diff
@@ -49,7 +50,11 @@
                 <action android:name="android.intent.action.VIEW" />
                 <category android:name="android.intent.category.DEFAULT" />
                 <category android:name="android.intent.category.BROWSABLE" />
-                <data android:scheme="openwallet" android:host="callback" />
+                <data android:scheme="openwallet"/>
+                <data android:host="callback"/>
+                <!-- credential offer / haip -->
```

Comment:
Do you need to include haip scheme as well? If not for some reason, could you add a comment?

Reply:
Not sure if haip scheme is any different? I'll add a comment 👍